addons/web/static/tests/views/fields/one2many_field_tests.js

```diff
@@ -2510,6 +2510,39 @@ QUnit.module("Fields", (hooks) => {
         await clickSave(target);
     });
 
+    QUnit.test("open a record in a one2many kanban with should display delete button", async function (assert) {
+        assert.expect(1);
+
+        await makeView({
+            type: "form",
+            resModel: "partner",
+            serverData,
+            arch: `
+                <form>
+                    <field name="turtles">
+                        <kanban>
+                            <field name="display_name"/>
+                            <templates>
+                                <t t-name="kanban-box">
+                                    <div t-att-class="'oe_kanban_global_click'">
+                                        <h3>Record 1</h3>
+                                    </div>
+                                </t>
+                            </templates>
+                        </kanban>
+                        <form>
+                            <field name="turtle_foo"/>
+                        </form>
```

Comment:
Why is there a form inside a form? This seems a bit unusual.

Reply:
I have tried to create the similar view same as res.partner view in testcase
https://github.com/odoo/odoo/blob/16.0/odoo/addons/base/views/res_partner_views.xml#L235